packages/fuchsia_ctl/pubspec.yaml

```diff
@@ -14,10 +14,11 @@ dependencies:
   file: ^5.0.10
   meta: ^1.1.7
   path: ^1.6.4
+  pedantic: ^1.9.2
```

Comment:
Please avoid having pedantic as a direct dependency - use it as a dev dependency instead. Although this package is not consumed by anyone, this can cause problems if we later need to use some other package that's also using pedantic with an incompatible constraint. Pedantic's semver constraints are really more about the analysis option rules anyway, rather than the `unawaited` function.

We should just write our own `unawaited` function, which is literally one line of code, where we need it.

Reply:
done